### PR TITLE
Add display_name to API response

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
@sarah256 feel free to review this :smiley:

**Comment by mprahl** [comment]
@yashvardhannanavati could you please fix the tests?

File: tests/api/test_all_stories.py [hunk 1]
```diff
@@ -95,9 +99,9 @@
                 }
             ],
             'meta':{
+                'requested_node_index': 0,
                 'story_related_nodes_backward': [0, 0, 0, 1],
-                'story_related_nodes_forward': [0, 1, 0, 0],
-                'requested_node_index': 0
+                'story_related_nodes_forward':[0, 1, 0, 0]
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 2]
```diff
@@ -95,9 +99,9 @@
                 }
             ],
             'meta':{
+                'requested_node_index': 0,
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 3]
```diff
@@ -204,9 +214,9 @@
                 }
             ],
             'meta': {
+                'requested_node_index': 0,
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 4]
```diff
@@ -204,9 +214,9 @@
...
                 'story_related_nodes_backward': [0, 0, 0, 1, 0, 1],
-                'story_related_nodes_forward': [0, 1, 0, 0, 0, 0],
-                'requested_node_index': 0
+                'story_related_nodes_forward':[0, 1, 0, 0, 0, 0]
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 5]
```diff
@@ -324,16 +338,17 @@
                 }
             ],
             'meta':{
-                'story_related_nodes_forward': [0, 1, 0, 0],
+                'requested_node_index': 2,
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 6]
```diff
@@ -324,16 +338,17 @@
...
                 'story_related_nodes_backward': [0, 0, 0, 1],
-                'requested_node_index': 2
+                'story_related_nodes_forward':[0, 1, 0, 0]
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 7]
```diff
@@ -442,16 +460,17 @@
                 }
             ],
             'meta':{
-                'story_related_nodes_forward': [0, 1, 0, 0],
+                'requested_node_index': 2,
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 8]
```diff
@@ -442,16 +460,17 @@
...
                 'story_related_nodes_backward': [0, 0, 0, 1],
-                'requested_node_index': 2
+                'story_related_nodes_forward':[0, 1, 0, 0]
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 9]
```diff
@@ -584,16 +608,17 @@
                 }
             ],
             'meta': {
-                'story_related_nodes_forward': [0, 1, 0, 0, 0, 0],
+                'requested_node_index': 2,
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 10]
```diff
@@ -584,16 +608,17 @@
...
                 'story_related_nodes_backward': [0, 0, 0, 1, 0, 1],
-                'requested_node_index': 2
+                'story_related_nodes_forward':[0, 1, 0, 0, 0, 0]
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 11]
```diff
@@ -726,9 +756,9 @@
                 }
             ],
             'meta': {
-                'story_related_nodes_forward': [0, 1, 0, 0, 0, 0],
+                'requested_node_index': 2,
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_all_stories.py [hunk 12]
```diff
@@ -726,9 +756,9 @@
...
                 'story_related_nodes_backward': [0, 0, 0, 1, 0, 1],
-                'requested_node_index': 2
+                'story_related_nodes_forward':[0, 1, 0, 0, 0, 0]
```
Comment (mprahl) [inline tests/api/test_all_stories.py]: This change seems unnecessary.

File: tests/api/test_story.py [hunk 13]
```diff
@@ -113,7 +118,9 @@
                 'original_nvr': None,
                 'release': '4.el7_4_as',
                 'resource_type': 'ContainerKojiBuild',
+                'display_name': 'slf4j_2-1.7.4-4.el7_4_as',
                 'start_time': '2017-04-02T19:39:06+00:00',
+
```
Comment (mprahl) [inline tests/api/test_story.py]: This change seems unnecessary.

File: tests/api/test_story.py [hunk 14]
```diff
@@ -877,15 +920,16 @@
         ],
         'meta':{
             'requested_node_index': 5,
-            'story_related_nodes_forward': [1, 0, 0, 0, 0, 0, 0],
-            'story_related_nodes_backward': [0, 0, 0, 0, 0, 0, 0]
+            'story_related_nodes_backward': [0, 0, 0, 0, 0, 0, 0],
```
Comment (mprahl) [inline tests/api/test_story.py]: This change seems unnecessary.

File: tests/api/test_story.py [hunk 15]
```diff
@@ -877,15 +920,16 @@
...
+            'story_related_nodes_forward':[1, 0, 0, 0, 0, 0, 0]
```
Comment (mprahl) [inline tests/api/test_story.py]: This change seems unnecessary.

File: tests/api/test_story.py [hunk 16]
```diff
@@ -1009,8 +1060,8 @@
         ],
         'meta':{
             'requested_node_index': 6,
-            'story_related_nodes_forward': [1, 0, 0, 0, 0, 0, 0],
-            'story_related_nodes_backward': [0, 0, 0, 0, 0, 0, 0]
+            'story_related_nodes_backward': [0, 0, 0, 0, 0, 0, 0],
```
Comment (mprahl) [inline tests/api/test_story.py]: This change seems unnecessary.

File: tests/api/test_story.py [hunk 17]
```diff
@@ -1009,8 +1060,8 @@
...
+            'story_related_nodes_forward':[1, 0, 0, 0, 0, 0, 0]
```
Comment (mprahl) [inline tests/api/test_story.py]: This change seems unnecessary.

**Comment by mprahl** [comment]
It seems like there are a lot of changes in the tests in regards to the ordering of the expected meta dictionaries, but it doesn't actually alter the tests at all. If you're trying to order the keys alphabetically for readability, then I'll let it slide but ideally it should be in a separate commit.

File: tests/api/test_all_stories.py (hunk 2 above)
Comment (yashvardhannanavati) [reply tests/api/test_all_stories.py]: I put a pprint again and changed the expected output. Maybe that changed the order or something

**Comment by yashvardhannanavati** [comment]
@mprahl this is ready for review again :)

File: tests/api/test_story.py [hunk 18]
```diff
@@ -902,13 +945,15 @@
             {
                 'author_date': '2017-04-26T11:44:38+00:00',
                 'commit_date': '2017-04-26T11:44:38+00:00',
+                'display_name': 'commit #8a63adb',
                 'hash': '8a63adb248ba633e200067e1ad6dc61931727bad',
                 'log_message': 'Related: #12345 - fix xyz',
-                'resource_type': 'DistGitCommit'
+                'resource_type': 'DistGitCommit',
```
Comment (mprahl) [inline tests/api/test_story.py]: This change is unnecessary but I'll let it slide :)

**Comment by mprahl** [comment]
:+1:

**Pull request merged**